{"name": "test_1", "status": "failed", "statusDetails": {"message": "AssertionError: assert 100 == 50", "trace": "def test_1():\n        print('执行测试用例一')\n        # 预期结果\n        a=100\n        # 实际结果\n        b=50\n        # 判断实际结果与预期结果是否一致\n>       assert a == b\nE       assert 100 == 50\n\nv1\\test_a.py:14: AssertionError"}, "attachments": [{"name": "stdout", "source": "d8d16b36-0cbc-4fd0-b7cc-62bd34963599-attachment.txt", "type": "text/plain"}], "start": 1566632376046, "stop": 1566632376047, "uuid": "2be17353-b07f-4c4a-8c91-90a72f49e277", "historyId": "4e70ffa82fbe886e3c4ac00ac374c29b", "testCaseId": "9cb007cd62b2d9d784c367974654c188", "fullName": "v1.test_a#test_1", "labels": [{"name": "parentSuite", "value": "v1"}, {"name": "suite", "value": "test_a"}, {"name": "subSuite"}, {"name": "host", "value": "DESKTOP-S5STCAS"}, {"name": "thread", "value": "25408-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "v1.test_a"}]}